File: packages/sdk/sdk-py/examples/responses_simple.py
import os
from phaseo import Phaseo

api_key = os.getenv("PHASEO_API_KEY")
if not api_key:
	raise RuntimeError("Set PHASEO_API_KEY")

client = Phaseo(api_key=api_key)

response = client.responses.create({
	"model": "openai/gpt-5-nano",
	"input": "Write a one-sentence summary of why testing matters.",
})

print(response)

File: packages/sdk/sdk-ts/examples/responses-simple.ts
import { Phaseo } from "../src";

const apiKey = process.env.PHASEO_API_KEY;
if (!apiKey) throw new Error("Set PHASEO_API_KEY");

async function main() {
	const client = new Phaseo({ apiKey });
	const response = await client.responses.create({
		model: "openai/gpt-5-nano",
		input: "Write a one-sentence summary of why testing matters.",
	});

	console.log(JSON.stringify(response, null, 2));
}

main().catch((err) => {
	console.error(err);
	process.exitCode = 1;
});
